Parser/Token.cs

```diff
@@ -37,9 +37,24 @@ public enum E {
 
    // Utility function used to echo an error to the console
    public void PrintError () {
+      Console.OutputEncoding = Encoding.Unicode;
       if (Kind != ERROR) throw new Exception ("PrintError called on a non-error token");
       Console.ForegroundColor = ConsoleColor.Yellow;

```

Comment:
See the sample image I sent you - only the actual error text is in yellow, the source code is printed in white.

Reply:
Corrected to print only the error text in yellow